src/components/TodoList.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Todo } from '../types/types';
+
+import { TodoInfo } from './TodoInfo';
+import { UserInfo } from './UserInfo';
+
+export const TodoList: React.FC<{ todos:Todo[] }> = ({ todos }) => (
+  <>
+    <ul className="todo__list">
+      {todos.map((todo) => (
+        <li className="todo__item">
+          <UserInfo {...todo} />
+          <TodoInfo {...todo} />
```

Comment:
не логічно передавати в userInfo якусь тудушку. Також не потрібно використовувати рест а в компоненті збирати все назад спредом. Це геть не логічно
```suggestion
          <UserInfo user={todo.user} />
          <TodoInfo todo={todo} />
```